### ISSUE TITLE
Continuous query misleading logic

### ISSUE THREAD

**Issue by arturasjurev** [influxdata/influxdb#10189]
__System info:__ [Include InfluxDB version, operating system name, and other relevant details]
Ubuntu Server Bionic Beaver 18.04
InfluxDB shell version: 1.6.0

__Steps to reproduce:__

1. Execute this in influx:
`> > CREATE CONTINUOUS QUERY CQ_tx_bytes__to_tx_speed_1594367952 ON "x-org_1594367952" RESAMPLE EVERY 5m BEGIN SELECT non_negative_derivative(value, 8s) AS value INTO "x-org_1594367952".RP_1h.tx_speed FROM "x-org_1594367952".RP_1h.tx_bytes_ WHERE time > now() - 1h GROUP BY * END
`

__Expected behavior:__ [What you expected to happen]
Expecting continuous query to be created.

__Actual behavior:__ [What actually happened]
`
ERR: error parsing query: found *, expected GROUP BY time(...) at line 1, char 294
`
Yet if I do what's inside of CQ creation query, I get wanted output.

__Additional info:__ [Include gist of relevant config, logs, etc.]
If I for example execute : 
`
CREATE CONTINUOUS QUERY CQ_rx_bytes_to_rx_speed_1594367952 ON "x-org-clients_1594367952" RESAMPLE EVERY 5m BEGIN SELECT non_negative_derivative(value, 8s) AS value INTO "x-org-clients_1594367952".RP_1h.rx_speed FROM "x-org-clients_1594367952".RP_1h.rx_bytes WHERE time > now() - 1h GROUP BY time(5m), * END
`
This doesn't do anything in meantime.

Basically query `SELECT non_negative_derivative(value, 8s) AS value INTO "x-org_1594367952".RP_1h.tx_speed FROM "x-org_1594367952".RP_1h.tx_bytes_ WHERE time > now() - 1h GROUP BY *` is what I want, but it cannot be created in CQ.

**Comment by arturasjurev** [comment]
Also if I do:
`> SELECT non_negative_derivative(value, 8s) AS value INTO "x-org-clients_1594367952".RP_1h.ucast_tx_packets_per_time FROM "x-org-clients_1594367952".RP_1h.ucast_tx_packets WHERE time > now() - 1h GROUP BY time(5m)`
I get :
`ERR: aggregate function required inside the call to non_negative_derivative`
Maybe it's my lack of knowledge but I cant find solution to this.
